dvc/command/data_sync.py

```diff
@@ -116,7 +116,7 @@ def shared_parent_parser():
         "targets",
         nargs="*",
         default=None,
-        help="Limit the scope to these stage files.",
+        help="Limit the scope to these DVC-files. With -R a directory to search DVC-files in can be specified.",
```

Comment:
please, check the contributing guide and make sure that you run `black` manually or via the pre-commit hook to enforce the style guide.